### ISSUE TITLE
Invalid header X-aws-ec2-metadata-token (invalid return char or leading space in header)

### ISSUE THREAD

**Issue by omarryhan** [sjkingo/django-ebhealthcheck#4]
Hey,

Thanks for the awesome work you've put into this lib!
In our CI (Github actions), we're occasionally getting this error when `get_ec2_instance_ip()` runs.

```
requests.exceptions.InvalidHeader: Invalid return character or leading space in header: X-aws-ec2-metadata-token
```

It happens approximately 1 in 4 times everytime our CI runs. Usually when we face this issue, we just manually run the CI in hope of a metadata-token that won't throw this error, but it's getting a bit tedious. Especially that our CI runs more than once a day.

I'll work on a fix and create a PR now.

**Comment by DataGreed** [comment]
I face a similar issue, it fails almost every time for me when I run it in github actions. Weirdly enough, it works well locally.

@sjkingo please consider the pull request

**Comment by omarryhan** [comment]
Here's my solution @DataGreed 
https://github.com/sjkingo/django-ebhealthcheck/pull/5#issuecomment-1151616127

**Comment by sjkingo** [comment]
Thanks for the patch, I've merged this now and released as 2.0.2.

Sorry for the delay!

**Comment by DataGreed** [comment]
@omarryhan  @sjkingo thank you so much 👍
